src/Features/CSharp/Portable/CSharpFeaturesResources.resx

```diff
@@ -524,4 +524,7 @@
   <data name="Convert_for_to_foreach" xml:space="preserve">
     <value>Convert 'for' to 'foreach'</value>
   </data>
+  <data name="Allow_unsafe_code_in_this_project" xml:space="preserve">
+    <value>Allow unsafe code in this project</value>
```

Comment:
based on this string (which i think is good), i think teh feature would be called "UpdateProjectToAllowUnsafeCode

Reply:
#fixed